src/lidar/lidar_metadata_in_db.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct  5 16:34:48 2021.
+
+@author: pkh35
+"""
+
+import geoapis.lidar
+import json
+import geopandas as gpd
+import os
+import zipfile
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from geoalchemy2 import Geometry
+from sqlalchemy.orm import sessionmaker
+from src.digitaltwin import setup_environment
+
+Base = declarative_base()
+
+
+class Lidar(Base):
+    """Create lidar table in the database."""
+
+    __tablename__ = 'lidar'
+    unique_id = Column(Integer, primary_key=True, autoincrement=True)
+    filepath = Column(String)
+    Filename = Column(String)
+    filename_no_format = Column(String)
+    geometry = Column(Geometry('POLYGON'))
+
+
+engine = setup_environment.get_database()
+Lidar.__table__.create(bind=engine, checkfirst=True)
+
+
+def get_files(filetype, folder):
+    """To get the path of the downloaded point cloud files."""
+    file_list = []
+    files_path = []
+    for (paths, dirs, files) in os.walk(folder):
+        for file in files:
+            if file.endswith(filetype):
+                file_list.append(os.path.join(paths, file))
+    for filepath in file_list:
+        filepath = filepath.replace(os.sep, '/')
+        filepath = "'{}'".format(filepath)
+        filepath = filepath.replace("'", "")
+        files_path.append(filepath)
+    return files_path
+
+
+def store_lidar_path(filetype, folder):
+    """To store the path of downloaded point cloud data."""
+    laz_files = get_files(filetype, folder)
+    for file in laz_files:
+        file_name = os.path.basename(file)
+        file_name = file_name.replace("'", "")
+        file_name_no_format = file_name.rsplit('.', 1)[0]
+        lidar = Lidar(filepath=file, Filename=file_name,
+                      filename_no_format=file_name_no_format)
+        Session = sessionmaker(bind=engine)
+        session = Session()
+        session.add(lidar)
+        session.commit()
+        remove_duplicate_rows("lidar")
+
+
+def remove_duplicate_rows(table_name):
+    """Remove duplicate rows from the tables."""
+    # add tbl_id column in each table
+    engine.execute('ALTER TABLE \"%(table_name)s\" ADD COLUMN IF NOT EXISTS tbl_id SERIAL' % (
+        {'table_name': table_name}))
```

Comment:
This could be in another function, since its a side effect people may not expect from this function

Reply:
Good point. Can be addressed in a later branch if it doesn't get refactored away!